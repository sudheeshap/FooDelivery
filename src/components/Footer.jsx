import React from 'react';

export default function Footer() {
  return (
    <footer>
      <ul className="footer__navbar">
        <li className="footer__nav">Link1</li>
        <li className="footer__nav">Link2</li>
        <li className="footer__nav">Link3</li>
      </ul>
    </footer>
  );
}
