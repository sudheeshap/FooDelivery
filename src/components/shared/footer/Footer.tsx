import React from 'react';

export default function Footer() {
  return (
    <footer>
      <ul className="footer__navbar">
        <li className="footer__nav">
          <a href="/" className="footer__nav-link">
            About us
          </a>
        </li>
        <li className="footer__nav">
          <a href="/" className="footer__nav-link">
            Become a partner
          </a>
        </li>
        <li className="footer__nav">
          <a href="/" className="footer__nav-link">
            Contact us
          </a>
        </li>
      </ul>
    </footer>
  );
}
