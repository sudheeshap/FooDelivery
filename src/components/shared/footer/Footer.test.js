import React from 'react';
import { mount } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Footer from './Footer';

describe('Footer', () => {
  it('renders the footer', () => {
    const footer = mount(<Footer />);

    expect(shallowToJson(footer)).toMatchSnapshot();
  });
});
