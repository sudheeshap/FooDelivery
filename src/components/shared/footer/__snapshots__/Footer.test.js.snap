// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders the footer 1`] = `
<Footer>
  <footer>
    <ul
      className="footer__navbar"
    >
      <li
        className="footer__nav"
      >
        <a
          className="footer__nav-link"
          href="/"
        >
          About us
        </a>
      </li>
      <li
        className="footer__nav"
      >
        <a
          className="footer__nav-link"
          href="/"
        >
          Become a partner
        </a>
      </li>
      <li
        className="footer__nav"
      >
        <a
          className="footer__nav-link"
          href="/"
        >
          Contact us
        </a>
      </li>
    </ul>
  </footer>
</Footer>
`;
